### PR TITLE
Feature: OPX config/show utilities

### PR DESCRIPTION
**Pull request by atanu-mandal**
Feature: OPX config/show utilities

### CODE REVIEW

**Comment by jeffyin-dell** [comment]
@atanu-mandal missing DCO sign-off

File: debian/control [hunk 1]
```diff
@@ -9,6 +9,6 @@ Vcs-Browser: https://github.com/open-switch/opx-tools
 Vcs-Git: https://github.com/open-switch/opx-tools.git
```
Comment (jeffyin-dell) [inline debian/control:9]: Should you also be defining additional Build-Depends?

**Review by jeffyin-dell — DISMISSED** [review]

**Comment by jeffyin-dell** [comment]
Approved, but PR needs signoff (will not merge without it) and please see my question about defining Build-depends in debian/control

File: debian/control (hunk 1 above)
Comment (atanu-mandal) [reply debian/control:9]: These are python scripts, it may not be needed.

**Review by jeffyin-dell — APPROVED** [review]

**Pull request merged**